internal/pkg/manifest/lb_web_svc.go

```diff
@@ -254,6 +254,10 @@ func (lbws *LoadBalancedWebService) ExposedPorts() ([]ExposedPort, error) {
 		exposedPorts = append(exposedPorts, out...)
 	}
 	exposedPorts = append(exposedPorts, lbws.RoutingRule.exposedPorts(exposedPorts, workloadName)...)
-
+	out, err := lbws.NLBConfig.exposedPorts(exposedPorts, workloadName)
```

Comment:
Do we need this for the other svc types like backend svc?

Reply:
We support NLB only for LBWS at the moment.